coveralls-ruby.gemspec

```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = '>= 1.8.7'
 
   gem.add_dependency 'json', '>= 1.8', '< 3'
-  gem.add_dependency 'simplecov', '~> 0.12.0'
-  gem.add_dependency 'tins', '~> 1.6'
-  gem.add_dependency 'term-ansicolor', '~> 1.3'
+  gem.add_dependency 'simplecov', '~> 0.10.0'
```

Comment:
I wonder if we really need to downgrade the dependency on `simplecov` to `~> 0.10.0`. I think that is a problem, because that version of `simplecov` doesn't work well with the lastest version of Ruby.

In fact there was just `simplecov 0.13.0` released that removes deprecation warnings on Ruby 2.4, but still works with 1.9.

Therefore IMHO `simplecov` should to be upgraded to `~> 0.13.0`, to support Ruby 2.4. 

Perhaps it is time to stop supporting Ruby 1.9 that reached end of life [two years ago](https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/)?